superset/databases/api.py

```diff
@@ -1524,11 +1539,61 @@ def import_(self) -> Response:
         command.run()
         return self.response(200, message="OK")
 
+    @expose("/csv_upload_metadata/", methods=("POST",))
+    @protect()
+    @statsd_metrics
+    @event_logger.log_this_with_context(
+        action=(
+            lambda self, *args, **kwargs: f"{self.__class__.__name__}"
+            ".csv_upload_metadata"
```

Comment:
if the above becomes `/csv_metadata` then we would need to change this to be `.csv_metadata.post` so as not to lose the `upload` context here.